src/main/java/org/opentripplanner/middleware/utils/ItineraryUtils.java

```diff
@@ -31,38 +29,23 @@
  */
 public class ItineraryUtils {
 
-    public static final String IGNORE_REALTIME_UPDATES_PARAM = "ignoreRealtimeUpdates";
-    public static final String DATE_PARAM = "date";
     public static final String MODE_PARAM = "mode";
```

Comment:
Doesn't appear to be used. Consider removing.

Reply:
Fixed in https://github.com/ibi-group/otp-middleware/pull/243/commits/340fc1f447e0d27adc41e5518d9229a8bdf0e061